queries.sql

```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon%';
```

Comment:
- [ ] Kindly follow the requirement for the query `Find all animals whose name ends in "mon"`.  Your query will find the animals whose name contains "mon".
### Hint:
```
remove '%' at the end of mon
```